test/chunk_info.c

```diff
@@ -544,14 +544,15 @@ test_get_chunk_info_highest_v18(hid_t fapl)
     /* Allocate input (compressed) buffer */
     inbuf = HDcalloc(1, z_dst_nbytes);
```

Comment:
Using calloc here quiets valgrind's 'uninitialized bytes' warning (1.12 used malloc), but still writes the larger, incorrect number of bytes.